package/MDAnalysis/core/topologyattrs.py

```diff
@@ -2250,20 +2250,14 @@ def _bondDict(self):
     def set_atoms(self, ag):
         return NotImplementedError("Cannot set bond information")
 
-    def get_atoms(self, ag, outside=True):
+    def get_atoms(self, ag):
         """
         Get subset for atoms.
 
         Parameters
         ----------
         ag : AtomGroup
-        outside : bool (optional)
-            Whether to include connections to atoms outside the given
-            AtomGroup.
 
-        .. versionchanged:: 1.1.0
-            Added the ``outside`` keyword. Set to ``True`` by default
-            to give the same behavior as previously
         """
         warn = True
```

Comment:
the warn variable was introduced to trigger the warn `if` path in the code that's being reverted, I guess it needs to go away too?

```suggestion
```